app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -241,17 +393,47 @@ export default function Navigation() {
             </div>
           </Hidden>
         </div>
-        <div className={classes.bug}>
-          <Hidden smDown>
-            <ExternalNavButton
-              route={handbookURL}
-              label={HELP}
-              labelVariant="h3"
-            />
-            <NavButton route={logoutRoute} label={LOG_OUT} />
-          </Hidden>
-          <ReportButton />
-        </div>
+
+        <Hidden smDown>
+          <div ref={menuRef} style={{ display: "flex", flexDirection: "row" }}>
+            <div
+              style={{ marginRight: 16, display: "flex", alignItems: "center" }}
+            >
+              <ReportButton />
+            </div>
+            <div
+              className={classes.menuBtn}
+              onClick={menuOpen ? handleMenuClose : handleMenuOpen}
+            >
+              <MenuIcon />
+              <Avatar src={user?.avatarUrl} className={classes.avatar}>
+                {user?.avatarUrl && user.avatarUrl?.length > 0
+                  ? null
+                  : acronym(user?.name)}
+              </Avatar>
```

Comment:
Looking at how this is used in other places, this logic is unnecessary since it's dealt with by MUI already if the user doesn't have an image if you use our Avatar component. You should be able to do this instead:

```suggestion
              <Avatar user={user} className={classes.avatar} />
```